ServerPort: 8090
AuboIP: '192.168.1.115'
EE_DIS_TCP: 0.13
TBC: [0.05974 , 0.05690 , 0.99659,  0.29552826,
      -0.99807, -0.01375, 0.06061, -0.29920932,
      0.01715 , -0.99829, 0.05597,  0.64820029,
      0,0,0,1 ]
CalibrationPoint: [15.17,58.218,158.196,102.856,74.259,-91.6399]
ViewPointLowestPoint: [23.033,-71.261,143.8354,158.669,77.05,-75.4]
ViewPointFrontStartPoint: [-4.56702,64.6972,129.2913,79.8718,102.09,-88.3972]
CalibrationPointXYZ_IN_TCP: [0.502288,-0.144040,0.527526]

Aubomaxacctuple: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
Aubomaxvelctuple: [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
Aubominacctuple:  [0.1,0.1,0.1,0.1,0.1,0.1]
Aubominvelctuple: [0.1,0.1,0.1,0.1,0.1,0.1]